tests/tests.py

```diff
@@ -162,56 +167,53 @@ def test_stats_last_month(self):
         args = parse_args(['stats', '--last-month'])
         args.func(args)
         output = sys.stdout.getvalue().strip()
-        self.assertEqual(output, "Work: 06h 00min\nSlack: 02h 40min")
+        self.assertEqual(output, "Work: 06h 00m\nSlack: 02h 40m")
 
     def test_stats_from(self):
         output = self.mock_date_and_stdout(
             ['stats', '--from', '2015-01-02'],
             date_value=datetime.datetime(2015, 1, 5)
         )
-        self.assertEqual(output, "Work: 03h 10min\nSlack: 01h 30min")
+        self.assertEqual(output, "Work: 03h 10m\nSlack: 01h 30m")
```

Comment:
hah! thanks for fixing all the tests I broke. I was going to do that at some point :D